README.md

```diff
@@ -1104,6 +1105,48 @@ bash -v scriptname
 bash -x scriptname
 ```
 
+# 5. Package Management
+There are two types of packages available for linux systems.
```

Comment:
I think it would be worth to mention that some linux distributions don't use these types of packages (like Arch Linux for instance)